tests/ImplicitSpec.hs

```diff
@@ -35,7 +35,7 @@ import Test.QuickCheck
       forAll)
 import Data.Foldable ( for_ )
 import Test.Hspec.QuickCheck (prop)
-import Linear (V2(V2), V3(V3), V4(V4), (^*) , Epsilon(nearZero))
```

Comment:
isn't removing this going to cause problems with values really close to zero?

Reply:
Wasn't able to hit a problem with it even with quite large `--qc-max-success`. It was only added recently to fix disabled  `scaling` property in https://github.com/Haskell-Things/ImplicitCAD/pull/410/commits/7eff263a0f2bc7e307434211ba2003146f0a4c01#diff-cb79804fa5faf09727f9120b83f037c50a213c44faf3058e604a42fccc967655L138-R143 but this is more elegant since we can just use `Eq` now.